### DISCUSSION TITLE
Repository README.md images cache not update

### DISCUSSION THREAD

**Discussion by TakWolf** [community/community#46773] [Repositories]
### Select Topic Area

Bug

### Body

This repository is:  https://github.com/TakWolf/retro-pixel-font

The picture referenced by README is not the original url, but the cache from  https://raw.githubusercontent.com

But it not update! It's been more than 24 hours. I tried some ways(like update a new image), but it didn't work.

The following images are the same nodes.

https://github.com/TakWolf/retro-pixel-font/blob/master/docs/readme-banner.png
<img width="1268" alt="image" src="https://user-images.githubusercontent.com/6064962/217621766-ad8c709d-d146-4f6b-a46f-8007ef5ec583.png">
<img width="1237" alt="image" src="https://user-images.githubusercontent.com/6064962/217621870-e9ca414e-191e-46b5-ad7f-2ebf3e897030.png">

https://github.com/TakWolf/retro-pixel-font/blob/master/docs/thick/preview.png
<img width="1290" alt="image" src="https://user-images.githubusercontent.com/6064962/217622180-76f9b381-4288-4487-9cff-94f7011314e2.png">
<img width="1179" alt="image" src="https://user-images.githubusercontent.com/6064962/217622280-36c5712b-691d-4795-9933-ec08d8c3dda2.png">

**Comment by 2Abendsegler** [comment]
I have the same problem. The raw.githubusercontent.com links show content from 48 hours ago. I [reported a bug](https://support.github.com/contact/bug-report) about 34 hours ago. However, [the ticket for it](https://support.github.com/ticket/personal/0/2005506) is probably not visible to everyone. There are no results yet.

[Here](https://github.com/orgs/community/discussions/46691) and [here](https://github.com/orgs/community/discussions/46758) are more forks with the same problem.

**Comment by ThePat02** [comment]
Same for me in my [repo](https://github.com/ThePat02/SplashScreenWizard).

**Comment by abdelrahmanhatemdev** [comment]
Same for me in my [repo](https://github.com/abdulrahmanhatem/wall-clock)

**Comment by cloydlau** [comment]
Same problem

**Comment by leonardo-minaya** [comment]
I found the solution [here](https://github.com/atom/markdown-preview/issues/207#issuecomment-248848108). It worked for me

**Comment by naitikmalaviya** [comment]
Just in case someone ends up here, purging the cache is also an option. Read more [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls) and [here](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls).

**Example:**
```
$ curl -X PURGE https://<subdomain>.githubusercontent.com/4d04abe0044d94fefcf9af2133223....
```

**Reply by yushine** [reply]
It works for my repo https://github.com/innocommerce/innoshop, thanks.
